test/heroes.test.js

```diff
@@ -0,0 +1,96 @@
+var chai = require('chai'),
```

Comment:
I'd like to see the controller using some fake data and not manipulating the original one

Reply:
Every test manipulates one fake record: {"id":1, "name":"superman"}. Maybe it's not enough but that's more a matter of "needs more tests". Anyway' I'll merge.